R/Raylib/bundled/patches/add-missing-header.patch

```diff
@@ -0,0 +1,37 @@
+--- a/Makefile
++++ b/Makefile
+@@ -584,16 +584,14 @@ raudio.o : raudio.c raylib.h
+ raygui.o : raygui.c
+ 	$(CC) -c $< $(CFLAGS) $(INCLUDE_PATHS) -D$(PLATFORM) -DRAYGUI_IMPLEMENTATION
+ raygui.c:
+-	echo #define RAYGUI_IMPLEMENTATION > raygui.c
+-	echo #include "$(RAYLIB_MODULE_RAYGUI_PATH)/raygui.h" >> raygui.c
++	@echo '#include "extras/raygui.h"' > raygui.c
```

Comment:
If I understand this correctly, in the file you have `#include "extras/raygui.h"`, but then you install the header file directly in `${includedir}`?

Reply:
Yes. There are both implementation and declaration in `extras/raygui.h` (and also `extras/physac.h`). According to the origin logic of the Makefile, when the module is enable (`RAYLIB_MODULE_RAYGUI=TRUE`), it will write a .c file with single include and compile the object file. After that, it will install them in `${includedir}`, as [specified in the Makefile](https://github.com/raysan5/raylib/blob/master/src/Makefile#L589). 

I'm not sure if the header file is actually not needed here.